nautobot/apps/forms.py

```diff
@@ -0,0 +1,52 @@
+"""Forms and fields for apps to use."""
+
+from nautobot.extras.forms import (
+    CustomFieldModelBulkEditFormMixin,
+    CustomFieldModelCSVForm,
+    CustomFieldModelFormMixin,
+    NautobotBulkEditForm,
+    NautobotModelForm,
```

Comment:
Should `NautobotFilterForm` be exposed in `nautobot.apps`?

Reply:
My thinking was that it's "quasi-obsolete" now that we have dynamic filter UI, but maybe I'm looking too far into the future?

Reply:
Probably better to leave it out and add it if we find that we still need to use it. Don't want to create unnecessary breaking changes.